mass_mailing_list_dynamic/models/mail_mass_mailing_list.py

```diff
@@ -47,18 +46,36 @@ def action_sync(self):
             desired_partners = Partner.search(sync_domain)
             # Remove undesired contacts when synchronization is full
             if one.sync_method == "full":
-                Contact.search([
-                    ("list_id", "=", one.id),
-                    ("partner_id", "not in", desired_partners.ids),
-                ]).unlink()
-            current_contacts = Contact.search([("list_id", "=", one.id)])
-            current_partners = current_contacts.mapped("partner_id")
+                contacts = Contact.search([("list_ids", "in", [one.id]),
```

Comment:
Why changing the style in this search? This need one extra line for the same arguments.

Reply:
you're right, it's not necessary!